modules/gitops-argo-cd-installation.adoc

```diff
@@ -23,4 +23,16 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 .. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and check *Enabled*.  
 
-. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
\ No newline at end of file
+. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
++
+[NOTE]
+====
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
+====
+. To log in with your username and password, obtain the password for the Argo CD instance:
+.. In the left panel of the console, use the perspective switcher to switch to the *Developer* perspective.
+.. Use the *Project* drop-down list and select the name of the Argo CD instance, <Argo_CD_instance_name>, in the `openshift-gitops` project.
+.. Use the left navigation panel to navigate to the *Secrets* page.
+.. Select the *openshift-gitops-cluster* instance to display the password.
+.. Copy the password.
```

Comment:
Is this meant for a user defined ArgoCD or the default openshift-gitops instance? In any case I think the better way would be something like this:

```suggestion
.. Use the left navigation panel to navigate to the *Secrets* page.
.. Use the *Project* drop-down list and select the namespace of the Argo CD instance
.. Select the *<Argo_CD_instance_name>-cluster* instance to display the password.
.. Copy the password.
```

Reply:
As far as I know it is for user defined additional Argo CD instances. Please validate the changes.

Reply:
Also, should this term <Argo_CD_instance_name>-cluster be <Argo_CD_instance_name-cluster> instead, @varshab1210??

Reply:
No, **-cluster** is not a variable